From fb770521fe7fddfefd65ab39350d360bd8d17322 Mon Sep 17 00:00:00 2001
From: Xiaowei Bao <xiaowei.bao@nxp.com>
Date: Fri, 11 Jan 2019 15:06:37 +0800
Subject: [PATCH 023/235] Revert "ARMv8: dts: ls1046a: add the property of IB
 and OB"

This reverts commit c513346d8fe4c9592fe414a6f7afbc276c114c94.
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
index eb4aa51607d8..60234ca46fd7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
@@ -651,8 +651,6 @@
 			device_type = "pci";
 			dma-coherent;
 			num-lanes = <4>;
-			num-ib-windows = <6>;
-			num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x40 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x40 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
@@ -678,8 +676,6 @@
 			device_type = "pci";
 			dma-coherent;
 			num-lanes = <2>;
-			num-ib-windows = <6>;
-			num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x48 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x48 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
@@ -705,8 +701,6 @@
 			device_type = "pci";
 			dma-coherent;
 			num-lanes = <2>;
-			num-ib-windows = <6>;
-			num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x50 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x50 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
-- 
2.17.1

